File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_29574.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest29574():
    auth_header = request.headers.get('Authorization', '')
    data = ' '.join(str(auth_header).split())
    return jsonify({'error': 'An internal error occurred'}), 500

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest29574.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest29574 {

    @PostMapping(path="/BenchmarkTest29574", consumes="multipart/form-data")
    public void BenchmarkTest29574(@RequestPart("multipart_field") String multipartField, HttpServletRequest request, HttpServletResponse response) throws Exception {
        response.sendError(500, "Internal error");
    }
}
